backend/src/utils/httpsigjs/parser.ts

```diff
@@ -241,7 +241,7 @@ export function parseRequest(request: Request, options?: Options): ParsedSignatu
 			parsed.params.headers = ['date']
 		}
 	} else {
-		parsed.params.headers = parsed.params.headers.split(' ')
+		parsed.params.headers = (parsed.params.headers as string).split(' ')
```

Comment:
Rather than casting here we should create a new `headers: string[]` variable that is assigned from the `parsed.params.headers` property.
Also that property should get a proper type above, e.g.

```
  interface ParsedRequest {
    scheme: string
    params: { headers?: string }
    signingString: string
  }
	const parsed: ParsedRequest = {
		scheme: authz === request.headers.get(HEADER.SIG) ? 'Signature' : '',
		params: {},
		signingString: '',
	}
```

Reply:
Done, please have a look and let me know if how I fixed it is ok 🙂